docs/source/reference/config.rst

```diff
@@ -1111,6 +1114,39 @@ Kueue configuration (optional).
 
 Name of the `local queue <https://kueue.sigs.k8s.io/docs/concepts/local_queue/>`_ to use for SkyPilot jobs.
 
+.. _config-yaml-kubernetes-dws:
+
+``kubernetes.dws``
+~~~~~~~~~~~~~~~~~~
+
+GKE DWS configuration (optional).
+
+Refer to :ref:`Using DWS on GKE <dws-on-gke>` for more details.
+
+``kubernetes.dws.enabled``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Whether to enable DWS (optional).
+
+When ``enabled`` is set to ``true``, SkyPilot will automatically enable the flex-start with queued provisioning mode if ``kubernetes.kueue.local_queue_name`` is set, otherwise it will use the flex-start mode.
```

Comment:
```suggestion
When ``enabled: true``, SkyPilot will automatically use DWS with flex-start mode. If ``kubernetes.kueue.local_queue_name`` is set, it will use flex-start with queued provisioning mode.
```